Fenced_Frames_Ads_Reporting.md

```diff
@@ -109,7 +109,7 @@ Note `window.fence` here is a new namespace for APIs that are only available fro
 
 ## registerAdBeacon
 
-A similar API was initially discussed here: https://github.com/WICG/turtledove/issues/99 for reporting clicks. The idea is that the buyer and seller side worklets are able to register a URL with the browser in their reportWin and reportResult APIs. A beacon will be sent to the registered URL when events are reported by the fenced frame via reportEvent.
+A similar API was initially discussed here: https://github.com/WICG/turtledove/issues/99 for reporting clicks. The idea is that the buyer and seller side worklets are able to register a URL with the browser in their reportWin and reportResult APIs. The registered URL's site is required to be attested for Protected Audience API. Please see [the Privacy Sandbox enrollment attestation model](https://github.com/privacysandbox/attestation#the-privacy-sandbox-enrollment-attestation-model). If the URL is attested for Protected Audience API, a beacon will be sent to the registered URL when events are reported by the fenced frame via reportEvent. Otherwise the beacon will not be sent. In the case of redirect, the redirected site is not checked for enrollment and attestation since the browser doesn't add any data directly to the redirect URL.  The initial reporting destination site is responsible for acting in accordance with its attestation if it decides to share any data via the redirect URL.
```

Comment:
I don't know what "site" means in the context of "registered URL's site".  Is this the origin or eTLD+1 or domain or hostname?  alternately we could clarify this like the [attestation doc](https://github.com/privacysandbox/attestation/blob/main/how-to-enroll.md#:~:text=site%20(scheme%2C%20eTLD%2B1)) does by saying "site (scheme, eTLD+1)"

Can we also change "via the redirect URL." to "via the redirect."